### ISSUE TITLE
ValueError: Input strings must be a multiple of 16 in length when usin AES, CBC_mode

### ISSUE THREAD

**Issue by marius-cristian** [dlitz/pycrypto#277]
I open a bmp file,
and convert and pad the bytes intro strings of length multiple of 16

i check len(plaintext)%16 ==0
i check in file blockalgo.py on line 244 before returning, if it satisfies the mentioned proprety.
it does. the error is still thrown.

searched in the project for the error ValueError: Input strings must be.
Can't find it anywhere.
my string length modulo block size is 0

string len 11944000 mod 16: 0

i get this cryptic error still.

line 243, in encrypt
    return self._cipher.encrypt(plaintext)
ValueError: Input strings must be a multiple of 16 in length

**Comment by danielfreire1** [comment]
you are probably using different encoding in the pycrypto encoding padding.

```python
len("abca") 
result 4

len("abcã".encode("utf-8"))
result 5

```
In this case the padding is 11 and not 12

**Comment by marius-cristian** [comment]
Nope. I cycled through the padding space from +0 to +15. The same error
everytime.

**Comment by alxgross** [comment]
arrrrrggg. having the same issue and no answer here
